CONTRIBUTING.md

```diff
@@ -26,29 +26,35 @@ To run tests scripts, you have to install:
 Before creating a contribution, make sure your code passes the following checks
 
 ```shell
-./scripts/test_forge.sh
-./scripts/test_cast.sh
```

Comment:
Why it is removed? Those scripts have some nice features like downloading correct Scarb and Cairo compiler versions.